Super heavy weapons (#14)
GameBrains/er-core: modules/special-rules/pages/death-ray.adoc

---

When you shoot with a {death-ray} weapon:

* For each {death-ray} shot, pick any target within range and line of sight, even a specific unit in a detachment.

> * For each {death-ray} shot, roll a D6. If you roll a 2+, the target suffers a hit regardless of Armour values and regardless of cover.

Reviewer: @IJW-Wartrader I'm curious about the way you've started these sentences. Are there units which get multiple Death Ray 'shots' from a single Death Ray 'weapon'? (And if so, are there similar cases for other SHWs?)

I ask because it looks a bit odd and redundant to follow "When you shoot with a {death-ray} weapon:" with "For each {death-ray} shot", twice. So I wonder something in particular drove you to construct it that way?

Author: It was because of the Titan weapon variants, where some of the options have multiple shots.

But the war engine weapon mount rules _should_ be enough to remove this text from the super-heavy weapons. From memory, it's in Macro-Weapon, Death Ray, and Mega Cannon.

Reviewer: What do you think to this, would it present any problem with rules interpretation?

First, replace "When you shoot with a {death-ray} weapon:" with "When you make a {death-ray} shot:"

And then:
```suggestion
. Pick any target within range and line of sight -- even a specific unit in a detachment.
. Roll a D6.
If you roll a 2+, the target suffers a hit regardless of its Armour value.
```
(I tweaked the end of the last sentence too, to make it more concise -- I don't think we need two 'regardless of' qualifiers.)

Also, have we made it crystal clear that 'weapon arc' also affects 'line of sight' (or actually 'line of fire' in the original text)? If not, then I think that we should either tighten that up or be clear in instances like this one along the lines of "any target within range, **arc** and line of sight/fire".

Finally, I should integrate the following lines, because right now we actually have a two-step procedure followed by a bullet which might look odd.

Author: > First, replace "When you shoot with a {death-ray} weapon:" with "When you make a {death-ray} shot:"

Looks good to me.

> Also, have we made it crystal clear that 'weapon arc' also affects 'line of sight' (or actually 'line of fire' in the original text)? If not, then I think that we should either tighten that up or be clear in instances like this one along the lines of "any target within range, **arc** and line of sight/fire".

Shooting with war engines currently says 'Each weapon mount on a war engine has a fire arc, and can only be used against targets within that fire arc.'.

I think adding 'arc' to all the SHW texts would be doubling up on that, but if you think the fire arc text needs to expanded let me know.

Reviewer: Yeah, I favour the more 'minimal' solution where possible. I think that in the weapon arc* rules we need to be clear that it also contrains the line of fire -- I'll keep that in mind when I edit that some more (I've started, but deviated to work through some of the simpler bits).

*I'd like to propose that we use 'weapon arcs' rather than 'weapon fire arcs' or 'fire arcs' -- partly because it avoids the shoot/fire duality, but mainly because it is more inclusive and works better for Close Combat Weapons and Ripper Tentacles that don't have a ranged element.

Reviewer: I pushed changes in line with what we discussed here, so I'll resolve this conversation.
